spec/README.md

```diff
@@ -84,42 +88,42 @@ Updates to this specification will not make any valid _message_ invalid.
 
 Updates to this specification will not remove any syntax provided in this version.
 
-Updates to this specification will not specify an error for any message
-that previously did not specify an error.
+Updates to this specification will not specify an _error_ for any _message_
+that previously did not specify an _error_.
 
-Updates to this specification will not specify the use of a fallback value for any message
-that previously did not specify a fallback value.
+Updates to this specification will not specify the use of a _fallback value_ for any _message_
+that previously did not specify a _fallback value_.
 
 Updates to this specification will not change the syntactical meaning
 of any syntax defined in this specification.
 
-Updates to this specification will not remove any functions defined in the default registry.
+Updates to this specification will not remove any _functions_ defined in the default function registry.
 
-Updates to this specification will not remove any options or option values
-defined in the default registry.
+Updates to this specification will not remove any _options_ or _option_ values
+defined in the default function registry.
 
```

Comment:
One additional item. We should add the following clause for clarity.

```suggestion

Updates to this specification may **deprecate** _functions_, _options_, and _operands_ defined in the default registry, as is standard practice in Unicode specifications. This is to indicate that the use of those _functions_, _options_, or _operands_ is discouraged. However, they are never removed, and implementations can continue to support them.
```

Reply:
We already say this (line 146), except we don't mention _operands_ in the stability policy and, of course, it's much less explanatory. Note that the stability policy is about what we *won't* do.

Making this change would be normative and require WG consensus, so I'll change the tags and remove fast-track until we discuss Monday.

Reply:
Sounds good

Reply:
Eh, this is a completely separate change from this PR. It really should be handled separately, but we're going to feature-creep it into this same change once again, yes?

Something like this was previously discussed here: https://github.com/unicode-org/message-format-wg/pull/883#discussion_r1751083540, and the spec text already includes this, which is in an `[!IMPORTANT]` block and therefore normative: https://github.com/unicode-org/message-format-wg/blob/f3344b5cea59e46aa8a0aa150904f53eb7690545/spec/README.md?plain=1#L142

Reply:
> Eh, this is a completely separate change from this PR. It really should be handled separately, but we're going to feature-creep it into this same change once again, yes?

Under normal circumstances, I would insist on it. But I'm hoping to be done tomorrow.